lib/daru/io/importers/sql.rb

```diff
@@ -55,28 +55,36 @@ def call
 
         private
 
-        def result_hash
-          column_names
-            .map(&:to_sym)
-            .zip(rows.transpose)
-            .to_h
+        def attempt_sqlite3_connection(db)
+          DBI.connect("DBI:SQLite3:#{db}")
+        rescue SQLite3::NotADatabaseException
+          raise ArgumentError, "Expected #{db} to point to a SQLite3 database"
+        rescue NameError
+          raise NameError, "In order to establish a connection to #{db}, please require 'dbi'"
```

Comment:
Now that `optional_gem 'dbi'` is already specified in the `#initialize` method, lines 62-63 are redundant and can be removed right?